model-optimizer/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -233,6 +233,21 @@ def assign_add_output_result(op: Node):
     op.out_port(0).connect(tmp_result.in_port(0))
 
 
+def topk_add_output_result(op: Node):
+    """
+    Function adds necessary output result node for TopK node
```

Comment:
Let's make the comment more detailed and describe all input/output values.

Reply:
Comments updeted.